CONTRIBUTING.md

```diff
@@ -0,0 +1,109 @@
+# Contributing
+
+Thank you for considering contributing to this project! We welcome contributions of all kinds. Here’s how you can help:
+
+## How to Get Started
+
+1. **Choose an Issue:**
+   - Browse through the [existing issues](https://github.com/move-fast-and-break-things/mfbt.community/issues) to find something you'd like to work on.
+   - If you find an issue you'd like to work on, please comment on it to let others know you're working on it.
+
+2. **Create an Issue:**
+   - If you don't find an existing issue that matches what you want to work on, you can [create a new issue](https://github.com/move-fast-and-break-things/mfbt.community/issues/new).
+   - Please provide as much detail as possible in the issue.
+
+## How to Submit Changes
+
+1. **Fork the Repository (if you don't have write access):**
+   - Click **"Fork"** on the repository page to create a copy in your account.
+
+2. **Clone the Repository:**
+   - If you have write access, clone the repository directly and create your branch from `main`.
+   - If you do not have write access, clone your fork and create your branch from `main`.
+
+3. **Work on the Issue:** Make your changes to the codebase.
+4. **Commit Your Changes:** Use descriptive commit messages that explain what changes you've made.
+5. **Push and Open a Pull Request:** Push your branch to your fork and open a pull request with a clear description of your changes.
+
+## How to Submit a Pull Request
+
+To ensure a smooth review process, please follow these guidelines when submitting a pull request:
+
+1. **Ensure Your Code is Up-to-Date:**
+   - Before starting your work, make sure to pull the latest changes from the `main` branch.
+   - If there are conflicts between your branch and `main`, resolve them locally before submitting your PR.
+
+2. **Branch Naming:**
+   - Use a descriptive branch name that reflects the changes.
+
+3. **Commit Messages:**
+   - Write clear and concise commit messages that explain the purpose of the changes.
+   - Use the present tense, e.g., "Add login feature" instead of "Added login feature."
```

Comment:
```suggestion
   - Use the present tense, e.g., "Add login feature" instead of "Added login feature."
   - Follow [the conventional commit specification](https://www.conventionalcommits.org/en/v1.0.0/)
```